model/registry/test.yaml

```diff
@@ -0,0 +1,106 @@
+groups:
+  - id: registry.test.suite
+    prefix: test.suite
+    type: attribute_group
+    brief: >
+     This group describes attributes specific to
+     [software tests](https://en.wikipedia.org/wiki/Software_testing).
+    attributes:
+      - id: name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable name of a [test suite](https://en.wikipedia.org/wiki/Test_suite).
+        examples: ['TestSuite1']
+      - id: run.id
+        type: string
+        stability: experimental
+        brief: >
+          The unique identifier of a test suite run.
+        examples: ['123408']
+      - id: run.status
+        type:
+          allow_custom_values: true
+          members:
+            - id: success
+              value: success
+              brief: success
+              stability: experimental
+            - id: failure
+              value: failure
+              brief: failure
+              stability: experimental
+            - id: skipped
+              value: skipped
+              brief: skipped
+              stability: experimental
+            - id: aborted
+              value: aborted
+              brief: aborted
+              stability: experimental
+            - id: timed_out
+              value: timed_out
+              brief: timed_out
+              stability: experimental
+        stability: experimental
+        brief: >
+          The status of the test suite run.
+        examples: ['success', 'failure', 'skipped', 'aborted', 'timed_out']
+      - id: case.name
+        type: string
+        stability: experimental
+        brief: >
+          The human readable name of the [test case](https://en.wikipedia.org/wiki/Test_case).
+        examples: ['TestCase1']
+      - id: case.run.id
+        type: string
+        stability: experimental
+        brief: >
+          The unique identifier of a test case run.
```

Comment:
Potentially. I think it would be odd for it to be different, and may not need to be set if `test.suite.run.id` is set, but it's possible to run a test case without running the entire suite. However, since a test case would still be within a suite (at least the way I'm thinking about it), one could argue this value is unnecessary.

Reply:
Resolving, can re-open back up if further discussion is needed.